save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -31,46 +29,35 @@ class Contest(
     @ManyToOne
     @JoinColumn(name = "organization_id")
     var organization: Organization,
-    var testSuiteIds: String = "",
```

Comment:
Can we have simple `@OneToMany var testSuites: List<TestSuite>` here?

Reply:
I am not sure how to map field in `Contest` entity and records in table `lnk_contest_test_suite`.
Seems like we need to store list of test suite ids in `contest` table, but as far as I remember mysql is unable to work with collections

Reply:
This does not require an test suite id in contest table. You can see how it is implemented @OneToMany in User entity

Reply:
Done